pori_python/ipr/constants.py

```diff
@@ -26,3 +26,16 @@
         "variantTypeName": "signature present",
     },
 }
+# Mapping hrd from pipeline terms to GraphKB terms
+HRD_MAPPING = {
+    "homologous recombination deficiency strong signature": {
+        "displayName": "homologous recombination deficiency strong signature",
+        "signatureName": "homologous recombination deficiency",
+        "variantTypeName": "strong signature",
+    },
+    "homologous recombination deficiency moderate signature": {
```

Comment:
Not sure what is the opposite of strong signature. Is it in case clininfo wants to match to moderate signatures?

Reply:
Unclear what they will want to do; the 'moderate' version exists in GKB currently but not as an opposite of strong. From Clininfo I am hearing that 'strong' should be seen as equivalent to 'present'. Maybe we should drop 'moderate'; will check with Clininfo.